jobs/utils.go

```diff
@@ -0,0 +1,38 @@
+package jobs
+
+import (
+	"context"
+
+	"api.audius.co/database"
+	"github.com/jackc/pgx/v5"
+)
+
+// How many tokens to fetch from the database in one batch.
+const tokenPageSize = 1000
+
+type ArtistCoin struct {
+	Mint string `db:"mint"`
+}
+
+func GetTokenCount(ctx context.Context, pool database.DbPool) (int, error) {
```

Comment:
both of these functions are not job utilities? they are very specific to these jobs. I don't think they belong in a utils file, and definitely shouldn't be publicly exported from here?